### PR TITLE
Etsy

### PR DESCRIPTION
**Pull request by Amaljose616**

### CODE REVIEW

File: etsy/index.html [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+            <a class="elements" href="">
+                <div class="fashion-image">
+                    <img src="assets/kitchen.webp" alt="">
+                </div>
+                <p>Kitchen & Dining</p>
+            </a>
+        </div>
+    </div>
+    <!-- <div class="popular">
+        <div class="popular-heading">
```
Comment (vonnue-akhil) [inline etsy/index.html:133]: Remove the comments if it's not necessary

File: etsy/style.css [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+.elements:hover img{
+    transform: scale(1.1);
+}
+.elements:hover p{
+    border-bottom: 2px solid #222222;
+}
+/* .first-image-section{
+    display: grid;
+    
+}
```
Comment (vonnue-akhil) [inline etsy/style.css]: remove if not necessary

**Review by vonnue-akhil — CHANGES_REQUESTED** [review]

**Comment by vonnue-akhil** [comment]
Add Screen Shots of the work

**Comment by Amaljose616** [comment]
](https://user-images.githubusercontent.com/115471075/202419597-dd50196a-b42c-4dcd-88f9-0d521ff08389.png)
2-8e59-492f-b6be-37ceeab0b179.png)

**Pull request merged**